### PR TITLE
fix: details icons

### PR DESCRIPTION
**Pull request by git-anurag-hub**
## Description

Fixes #(issue)

## Checklist

- [ ] My code follows the [contributing guidelines][contributing] of this project.
- [ ] I am aware that all my commits will be squashed into a single commit, using the PR title as the commit message.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have updated the user-facing documentation to describe any new or changed behavior.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have not reduced the existing code coverage.

## Comments

[contributing]: CONTRIBUTING.md

### CODE REVIEW

**Comment by git-anurag-hub** [comment]
> @git-anurag-hub passing the path would be a good idea this way since these `paths` will be relative, it might get confusing if we wanna store all our configs for components in one place because all the these components will be in different directories.

These are not relative path @JaeAeich but the URL of the SVG icon `package-author` wants to display.

File: packages/ecc-utils-design/src/components/details/details.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ interface FooterButton {
   key: string;
   name: string;
   variant?: "primary" | "success" | "neutral" | "warning" | "danger";
+  icon: string;
```
Comment (JaeAeich) [inline packages/ecc-utils-design/src/components/details/details.ts:30]: Should we make it optional like `variant`.

**Comment by git-anurag-hub** [comment]
Yeah that make sense!

**Review by JaeAeich — APPROVED** [review]

**Comment by JaeAeich** [comment]
@git-anurag-hub review and merge :+1:

**Pull request merged**